google/cloud/pubsub/internal/batching_publisher_connection.cc

```diff
@@ -178,6 +176,22 @@ void BatchingPublisherConnection::OnTimer() {
   FlushImpl(std::move(lk));
 }
 
+future<StatusOr<std::string>> BatchingPublisherConnection::CorkedError() {
+  promise<StatusOr<std::string>> p;
+  auto f = p.get_future();
+
+  struct MoveCapture {
+    promise<StatusOr<std::string>> p;
+    Status status;
+    void operator()() { p.set_value(std::move(status)); }
+  };
+  cq_.RunAsync(MoveCapture{std::move(p), corked_status_});
+
+  return f;
+}
+
+void Fits(std::unique_lock<std::mutex> const& lk, std::size_t bytes);
```

Comment:
Remnant of something?

Reply:
Yes, thanks!  I am surprised the compiler(s) did not catch that, thanks for pointing it out.